_data/harnesses/generate-test-data-streams/ksql.yml

```diff
@@ -184,6 +184,11 @@ prod:
           render:
             file: tutorials/generate-test-data-streams/ksql/markup/prod/submit-to-api.adoc
 
+        - action: sleep
+          ms: 10000
+          render:
+            skip: true
+
```

Comment:
Could this be (realtively easily) replaced with some kind of deterministic check similar to the existing `wait-for-containers.sh` script? The goal being toavoid the pitfalls of sleeps:
1. trying to find and manage over time the optimal wait time
2. making the user wait unecessarily

Reply:
The user doesn't experience this wait, it's only in Semaphore, and it's not the long pole.  IMO it's not worth the extra cycles to figure out the automated check for this one.